README.md

```diff
@@ -1,64 +1,45 @@
+The existence of several blockchain and distributed ledger technologies of different flavors in the market as well as networks of varying scopes and sizes built on them necessitates the need for interoperability and integration, lest we end up with a fragmented ecosystem where digital assets and the workfows (often contracts) governing them remain isolated in siloes. The solution to this is not to force all chains to coalesce (i.e., "*a single chain to rule them all*") but rather enable the networks to orchestrate transactions spanning their boundaries without sacrificing security, privacy, or governance autonomy (i.e., self-sovereignty). Hyperledger Cacti offers a family of protocols, modules, libraries, and SDKs, that can enable one network to be interoperable with, and carry out transactions directly with, another while eschewing the need for a central or common settlement chain. Cacti will allow networks to share ledger data, and exchange and transfer assets atomically, and manage identities, across their boundaries, as illustrated in the figure below.
+
+<img src="./images/cacti-vision.png">
 
-## Run the Examples
+As a fusion of two earlier systems (Cactus and Weaver) that have similar philosophies and goals, yet offer distinct mechanisms backed by differemt design and trust assumptions, Cacti offers a spectrum of seletable and configurable features for cross-network transaction orchestrations. An example illustrated below shows how distributed applications running on Fabric and Besu ledgers respectively can carry out the same set of cross-network transactions using the **Node Server** (Cactus legacy) or through **Relays** (Weaver legacy).
 
-### Supply Chain Example
+<img src="./images/tx-orchestration-modes.png">
 
-1. Ensure a working installation of [Docker](https://docs.docker.com/desktop/) is present on your machine.
-2. Run the following command to pull up the container that will run the example application and the test ledgers as well:
-    ```sh
-    docker run \
-      --rm \
-      --privileged \
-      -p 3000:3000 \
-      -p 3100:3100 \
-      -p 3200:3200 \
-      -p 4000:4000 \
-      -p 4100:4100 \
-      -p 4200:4200 \
-      ghcr.io/hyperledger/cactus-example-supply-chain-app:2022-04-05--feat-1579
-    ```
-3. Wait for the output to show the message `INFO (api-server): Cactus Cockpit reachable http://0.0.0.0:3200`
-4. Token generated by the application is displayed below
-5. Visit http://localhost:3200 in a web browser with Javascript enabled and insert the token when prompted
-6. Use the graphical user interface to create data on both ledgers and observe that a consistent view of the data from different ledgers is provided.
+## Roadmap
```

Comment:
@VRamakrishna Could you move these ROADMAP from the top to /ROADMAP.md? and remain only the link of [ROADMAP](https://github.com/hyperledger/cacti/blob/main/ROADMAP.md)?

I think we should overwrite the existing description on the current ROADMAP.md.

Reply:
Plus one!

Reply:
I retained the links in `README.md` but otherwise moved the section to `ROADMAP.md`. Please review.